build/Dockerfile

```diff
@@ -1,18 +1,16 @@
-FROM quay.io/operator-framework/ansible-operator:v0.17.0
+FROM quay.io/operator-framework/ansible-operator:v0.17.1
+USER root
 
 COPY requirements.yml ${HOME}/requirements.yml
-COPY image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
 RUN ansible-galaxy collection install -r ${HOME}/requirements.yml \
  && chmod -R ug+rwx ${HOME}/.ansible
 
-USER root
-RUN dnf install -y --nodocs redis --enablerepo=centos8-appstream && dnf clean all
-
-# Temporally workaround to fix https://github.com/operator-framework/operator-sdk/issues/2648
-RUN ln -s ${HOME}/.ansible /.ansible
```

Comment:
@mulbc I believe this line still needs to be in here. Log output from the CI system is showing it complaining because it cannot create the .ansible directory.

```
...stderr output: mkdir: cannot create directory ‘/.ansible’: Permission denied...
```